app/services/analytics_events.rb

```diff
@@ -1923,6 +1935,54 @@ def user_registration_2fa_setup_visit
     track_event('User Registration: 2FA Setup visited')
   end
 
+  # @param [String] redirect_from
+  # @param [Hash] vendor_status
+  # Tracks when vendor has outage
+  def vendor_outage(
+    redirect_from:,
+    vendor_status:,
+    **extra
+  )
+
+    track_event(
+      'Vendor Outage',
+      redirect_from: redirect_from,
+      vendor_status: vendor_status,
+      **extra,
+    )
+  end
+
+  # @param [Boolean] success
+  # @param [Integer] mfa_method_counts
+  # @param [Hash] pii_like_keypaths
```

Comment:
1. technically this is an array
2. we can skip documenting it, because it's not actually logged to the resulting event JSON: https://github.com/18F/identity-idp/blob/ba66ca9112a290158a6e378d456cf66f9f6d20f3/lib/analytics_events_documenter.rb#L13-L15
```suggestion
```